task_12/src/index.html

```diff
@@ -0,0 +1,277 @@
+<!doctype html>
+<html>
+<head>
+    <title>TASK 12, Responsive Layouts</title>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.css" rel="stylesheet">
+    <link href="css/index.css" rel="stylesheet">
+</head>
+<body>
+<header class="header">
+    <div class="header__wrapper">
+        <div class="header__timer">
+            <div class="header__watch">
+                XX:XX:XX
```

Comment:
Here must be present a semantic element `<time>`.

Reply:
Thank you, this issue was done.